judging-backend/main.cpp

```diff
@@ -33,25 +33,19 @@
 #define OLE 5
 #define CE 6
 #define IR 7
-#define RTE 0x08
-#define SEGV 0x10
-#define FPE 0x20
-#define ABRT 0x40
-#define DIS_SYS 0x80
+#define RTE 8
+#define SEGV 9
+#define FPE 10
+#define ABRT 11
+#define DIS_SYS 12
+#define ILL 13
 
 std::vector<std::string> input_files, output_files;
 
-void cleanse_string(std::string &str) {
-	// remove trailing whitespaces and newlines
-	while (str.back() == ' ' || str.back() == '\n') {
-		str.pop_back();
-	}
-}
-
 typedef bool (*func_ptr)(std::string &, std::string &);
```

Comment:
Unnecessary `std::string &` here. Consider using `const std::string&` (also modify `identical_checker`) instead.

Reply:
nits are the best thing invented by the programming industry